pkg/catalogv2/git/utils.go

```diff
@@ -0,0 +1,85 @@
+package git
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/pem"
+	"fmt"
+	"net/url"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+const (
+	stateDir             = "management-state/git-repo"
```

Comment:
Where did we get these values from? If it's defined somewhere in the code, maybe we should use that so this won't break if that changes

Reply:
Please resolve this as we discussed in private, this will be a new issue